plugin-server/src/utils/db/db.ts

```diff
@@ -515,6 +515,23 @@ export class DB {
         })
     }
 
+    public redisZRange(key: string, start: number, stop: number): Promise<string[]> {
+        return instrumentQuery(this.statsd, 'query.redisZRange', undefined, async () => {
+            const client = await this.redisPool.acquire()
+            const timeout = timeoutGuard('ZRANGE delayed. Waiting over 30 sec to perform ZRANGE', {
```

Comment:
It doesn't look like this actually times anything out, but just delays logging a message? Is there any way to set zrange to timeout?